Library/Homebrew/test/dev-cmd/install-bundler-gems_spec.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+require "cmd/shared_examples/args_parse"
+require "dev-cmd/install-bundler-gems"
+
+RSpec.describe Homebrew::DevCmd::InstallBundlerGems do
```

Comment:
Meta: might be nice to have a test some time for ensuring each command/dev-cmd has a test?